CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
```

Comment:
I realize we haven't discussed any of this. I'm a big fan of keeping a changelog and semantic versioning, so throwing this in here in the hope that there's no objection, but totally understand if you want to discuss it more. I can just take it out of this PR in that case.